eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/RichTextExtension.php

```diff
@@ -39,20 +39,20 @@ public function getFilters()
     {
         return array(
             new Twig_SimpleFilter(
-                'richtext_to_html5',
+                'ez_richtext_to_html5',
```

Comment:
No point in changing this, it's being dropped via #2659

Reply:
#2659 PR is still open, I can not taking it in account.